index.d.ts

```diff
@@ -0,0 +1,176 @@
+import * as React from 'react';
+import * as PIXI from 'pixi.js';
+
+declare module 'react-pixi-fiber' {
+
+  /**
+   * The set of object keys in T not in U.
+   *
+   * Attribution: https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766:
+   */
+  export type Diff<T extends string, U extends string> = ({ [P in T]: P } & { [P in U]: never } & { [x: string]: never })[T];
+
+  /**
+   * An object with keys in T not in U.
+   *
+   * Attribution: https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766:
+   */
+  export type Omit<T, K extends keyof T> = Pick<T, Diff<keyof T, K>>;
+
+  /** The structure of a `StagedComponent<P, S>` or `StagedStatlessComponent<P>`'s context. */
+  export interface StagedComponentContext {
+    app: PIXI.Application;
+  }
+
+  /** A `React.Component` with context configured and typed. */
+  export abstract class StagedComponent<P = {}, S = {}> extends React.Component<P, S> {
+    context: StagedComponentContext;
+  }
+
+  /** A `React.StatlessComponent<P>` with typed context.  To configure, use the `lift` or `connect` HoC's. */
+  export interface StagedStatlessComponent<P = {}> {
+    (props: P & { children?: React.ReactNode }, context: StagedComponentContext): React.ReactElement<any> | null;
+    propTypes?: React.ValidationMap<P>;
+    contextTypes?: React.ValidationMap<any>;
+    defaultProps?: Partial<P>;
+    displayName?: string;
+  }
+
+  /** A shorthand alias for `StagedStatelessComponent<P>`. */
+  export type StagedSFC<P = {}> = StagedStatlessComponent<P>;
+
+  /** `BitmapText` component properties. */
+  export interface BitmapTextProperties {
+    text?: string;
+  }
+
+  /**
+   * A component wrapper for `PIXI.extras.BitmapText`.
+   *
+   * see: http://pixijs.download/dev/docs/PIXI.extras.BitmapText.html
+   */
+  export class BitmapText extends React.Component<TilingSpriteProperties> {}
```

Comment:
I think `BitmapText` should not extend `TilingSpriteProperties`, but `Container`.